hwy/ops/ppc_vsx-inl.h

```diff
@@ -244,6 +244,8 @@ HWY_API VFromD<D> Undefined(D d) {
   // Suppressing maybe-uninitialized both here and at the caller does not work,
   // so initialize.
   return Zero(d);
+#elif HWY_HAS_BUILTIN(__builtin_nondeterministic_value)
```

Comment:
Should we move this condition to the first, just in case later GCC add this builtin?

Reply:
It is probably okay to leave __builtin_nondeterministic_value where it is as the __builtin_nondeterministic_value intrinsic is not available with GCC 14 or the GCC 15 trunk.